addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18677,7 +18677,7 @@ msgstr ""
 #. Description of setting "System -> Video output -> Gamma mode" with label #36572
 #: system/settings/settings.xml
 msgctxt "#36573"
-msgid "TODO: Gamma mode description"
+msgid "Select the gamma curve formula. Use BT.1886 for standard with technical 2.4 gamma that accounts for display black and white levels and avoids perceptual clipping. On displays with higher than zero black the effective gamma will be lower. Use input offset for a similar curve with a manually specified effective gamma. Output offset allows for perceptual clipping, but can be used to compensate for incorrect mastering monitor settings that lead to too bright dark details. Absolute gamma does not account for display black at all and will clip the lowest levels to display black on any display with higher than zero black level."
 msgstr ""
```

Comment:
I made it a bit more general. BT.1886 is a good default (as it is the current standard) - anybody choosing something different will hopefully know what they're doing. :)